src/analysis/retail/deathknight/shared/ExterminateCostNormalizer.ts

```diff
@@ -0,0 +1,73 @@
+import SPELLS from 'common/SPELLS';
+import talents from 'common/TALENTS/deathknight';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import { AnyEvent, CastEvent, HasAbility } from 'parser/core/Events';
```

Comment:
In case the EventType suggestions are applied.
```suggestion
import { AnyEvent, CastEvent, EventType, HasAbility } from 'parser/core/Events';
```